src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -391,70 +392,15 @@
     <EmbeddedResource Include="NetFX\ValueTuple\TupleElementNamesAttribute.cs" />
     <Content Include="SymbolsTests\CustomModifiers\GenericMethodWithModifiers.cpp" />
     <None Include="SymbolsTests\Metadata\public-and-private.cs" />
+    <Compile Include="ResourceLoader.cs" />
```

Comment:
Yep I opted out of globbing for this project and put it back as it was but I agree putting the compile items in a separate itemgroup is better. Will do that.